android/build.gradle

```diff
@@ -8,7 +8,7 @@ ext {
     PUBLISH_DESCRIPTION = 'Amplitude Kotlin client-side SDK for Android'
     PUBLISH_GROUP_ID = 'com.amplitude'
     PUBLISH_VERSION = '0.1.0'
-    PUBLISH_ARTIFACT_ID = 'amplitude-android'
+    PUBLISH_ARTIFACT_ID = 'amplitude-analytics-android-sdk'
```

Comment:
How about `analytics-android`? I agree with @bgiori the `amplitude` and `sdk` seem unnecessary.

Reply:
ok, let me update that for now. In that case, I will update SDK library to analytics-android as well